docs/src/app/components/CodeExample/index.js

```diff
@@ -40,9 +41,16 @@ class CodeExample extends React.Component {
       },
     };
 
+    const docs = parse(code);
```

Comment:
Yup, @nathanmarks and I talked about this before it was implemented. I like it, and it solves some other problems (string concat vs. backtick escaping). We can fix up the others when this merged. :+1: